### PR TITLE
Change menu layout

### PR DESCRIPTION
**Pull request by tomasmatus**

### CODE REVIEW

**Comment by tomasmatus** [comment]
So I think this looks somewhat good with the global menu. I'd say it's because the menu is "wide enough".

image

On the other hand it looks pretty funny when multiple files are selected. I think I'll need to set some minimal width which would cover this case

image

File: src/menu.tsx [hunk 1]
```diff
@@ -103,58 +121,86 @@ export function get_menu_items(
...
+                title: _("Copy"),
+                onClick: () => setClipboard([path + item.name]),
+                icon: <CopyIcon />,
+                description: _("Copy (Ctrl + C)"),
+            },
+            {
+                id: "paste-item",
+                title: _("Paste"),
+                isDisabled: clipboard.length === 0,
+                onClick: () => pasteFromClipboard(clipboard, cwdInfo, path, addAlert),
```
Comment (cockpituous) [inline src/menu.tsx:136]: This added line is not executed by any test.

File: src/menu.tsx [hunk 2]
```diff
@@ -103,58 +121,86 @@ export function get_menu_items(
...
                 title: _("Copy"),
                 onClick: () => setClipboard(selected.map(s => path + s.name)),
+                icon: <CopyIcon />,
+                description: _("Copy (Ctrl + C)"),
             },
+            {
+                id: "paste-item",
+                title: _("Paste"),
+                isDisabled: clipboard.length === 0,
+                onClick: () => pasteFromClipboard(clipboard, cwdInfo, path, addAlert),
```
Comment (cockpituous) [inline src/menu.tsx:197]: This added line is not executed by any test.

**Comment by jelly** [comment]
As discussed on Matrix, as we only have two icons we should not yet switch to an mixed icon/list style.

**Comment by tomasmatus** [comment]
Ok well then I don't understand what is there to work on here. Which is why I mentioned in the meeting last week that I didn't touch menus because it didn't make any sense to me why do anything until all the remaining features are added. Instead I was told to go forward with this... @jelly

**Comment by tomasmatus** [comment]
Closing as currently there are no needed changes for the menu and icons can be added later when remaining functionality is added (cut, undo & redo)

**Pull request closed without merging**